### DISCUSSION TITLE
Networking / OSC module?

### DISCUSSION THREAD

**Discussion by totalgee** [bjornbytes/lovr#572] [Q&A] [answered]
Hi there, this looks like a very cool project, especially for quick prototyping and trying things out, maybe even for live coding... I was wondering if there are any (third-party?) modules to do networking, in particular to send/receive UDP (e.g. [OSC](https://opensoundcontrol.stanford.edu/)) messages on the network (even just on `localhost`, to talk between LÖVR and other applications on the same machine)?

**Comment by lun-4** [comment, marked as answer]
https://lovr.org/docs/Plugins shows two networking related plugins, `lua-enet` and `lovr-luasocket` which might be useful for you!

**Reply by totalgee** [reply]
Great, thanks for the pointer, yes it looks like `lovr-luasocket` would do the trick (would still need to write the OSC message creation/parsing part).
